linux/newrelic-infra-centos.repo

```diff
@@ -0,0 +1,6 @@
+[newrelic-infra]
+name=New Relic Infrastructure
+baseurl=http://nr-downloads-main.s3-website-us-east-1.amazonaws.com/infrastructure_agent/linux/yum/el/8/x86_64
+gpgkey=http://nr-downloads-main.s3-website-us-east-1.amazonaws.com/infrastructure_agent/gpg/newrelic-infra.gpg
+gpgcheck=1
+repo_gpgcheck=1
```

Comment:
Since more `.repo` files can easily pop up, perhaps it's better to create a folder for them, e.g. `rpm-repos`